.github/workflows/main.yml

```diff
@@ -71,6 +71,15 @@ jobs:
           "s/node-and-date/no-local-version/g"
           setup.py
 
+      - name: Determine if it's a pre-release
+        id: check_prerelease
+        # run: echo ::set-output name=is_prerelease::${{ github.event.release.prerelease }}
+        # NOTE: For the sake of normal workflow, all releases are pre-releases.
+        #       The idea is that if the pre-release works, so does the regular release.
+        #       The other option would be to run for pre-release and regular release
+        #       but the trivial version for that would duplicate a bunch of the YAML.
+        run: echo ::set-output name=is_prerelease::true
```

Comment:
The `set-output` has been deprecated so pls consider using other options such as [GITHUB_ENV](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable). The same applies to line 96.